rockcraft.yaml

```
@@ -0,0 +1,117 @@
+name: fluent-bit
+summary: ROCK for the fluent-bit Project.
+description: This ROCK is a drop in replacement for the fluent/fluent-bit image.
+version: "2.1.6"
+license: Apache-2.0
+
+base: bare
+build-base: ubuntu@22.04
+#base: ubuntu@22.04
+platforms:
+  amd64:
+  arm64:
+
+entrypoint-service: fluent-bit
+services:
+  fluent-bit:
+    override: replace
+    summary: "fluent-bit service"
+    startup: enabled
+    command: "/fluent-bit/bin/fluent-bit [ -c /fluent-bit/etc/fluent-bit.conf ]"
+    on-failure: shutdown
+
+parts:
+  cmake:
```

```suggestion
  fluent-bit:
```
I think we are building fluent-bit here, so the part name of `cmake` does not make too much sense.